README.md

```diff
@@ -1 +1,151 @@
-# audit-service-deploy
\ No newline at end of file
+## Audit Service Deployment
+
+### Introduction
+
+The Audit Service is designed to provide auditing of user and system actions by defining the required events and the information associated with each event.  This definition is registered with the Audit service and used to create a Java SDK to record the audited events.  The Audit service is multi-tenant aware, which requires an application producing events to register each new tenant.  Applications send events using the generated Java SDK to Elasticsearch where they are Indexed according to tenant.
```

Comment:
"... to register each new tenant ..." - I know we previously registered tenants using the web service but this is no longer the case. 
"... they are Indexed according to tenant" - should this be re-worded to something like "they are indexed according to each tenant".